misk-aws-dynamodb/gradle.properties

```diff
@@ -0,0 +1,4 @@
+POM_ARTIFACT_ID=misk-aws-dynamodb
```

Comment:
do you have plans to migrate the existing usages?

Reply:
Yes.